src/components/Carousel/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC, ReactChild } from 'react';
+import { StyleSheet, View } from 'react-native';
+import Button from 'components/Button';

```

Comment:
for this specific project, don't use aliased import -- use relative path instead (i.e `../Button`)
will cause bug on library-consumer side with alias

Reply:
also, consider carefully when coupling different Components together, in this case.. Carousel depending on `Button` to work.